### PR TITLE
Add input redirection test

### PR DESCRIPTION
**Pull request by Diadlo**

### CODE REVIEW

File: t/feat_input_redirect [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+[ -e tests.sh ] && . ./tests.sh || . ../tests.sh
+
+echo "
+program=/bin/head
+arg1=-n2
+stdin=!python -c \"print('foo\nbar\nfoobar')\"
+" > /tmp/profile.rr2
```
Comment (radare) [inline t/feat_input_redirect]: dont do that, just commit the file in bins/*

dont assume /tmp exists

File: t/feat_input_redirect [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+NAME='rarun2 Input redirection'
+BROKEN=
+SHELLCMD='rarun2 /tmp/profile.rr2 | grep -q "^foo
+bar$" /tmp/out'
+EXITCODE=0
+run_test
+
+NAME='radare2 Input redirection'
+FILE='/bin/head'
+ARGS='-R /tmp/profile.rr2 -d'
```
Comment (radare) [inline t/feat_input_redirect]: this is not portable, so putit in the arch-specific debugger tests

File: t/feat_input_redirect [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+NAME='rarun2 Input redirection'
+BROKEN=
+SHELLCMD='rarun2 /tmp/profile.rr2 | grep -q "^foo
+bar$" /tmp/out'
+EXITCODE=0
+run_test
+
+NAME='radare2 Input redirection'
+FILE='/bin/head'
```
Comment (radare) [inline t/feat_input_redirect]: dont hardcode system binary paths /bin/head is not in any of my machines. commit it and put the test in the proper arch/os target

File: t/feat_input_redirect (hunk 1 above)
Comment (Diadlo) [inline t/feat_input_redirect]: Done.

File: t/feat_input_redirect (hunk 2 above)
Comment (Diadlo) [inline t/feat_input_redirect]: Done.

**Comment by Diadlo** [comment]
Done.

**Comment by Diadlo** [comment]
@radare ping :)

**Comment by radare** [comment]
yeah didnt forget about you :D

**Pull request merged**